app/controllers/map_controller.rb

```diff
@@ -65,19 +65,17 @@ def index
       {
         attribute: 'details',
         label: Store.human_attribute_name(:details),
-        type: 'text'
+        type: 'textarea'
       },
       {
         attribute: 'latitude',
         label: Store.human_attribute_name(:latitude),
-        type: 'text',
-        readonly: true
+        type: 'text'
```

Comment:
Why is this no longer read-only? Should we allow the users to put the latitude/longitude as they want instead of grabbing them from the map?

Reply:
Simao was trying to do this. So i think yes. Also if you copied it from gmaps it’s a good idea